### DISCUSSION TITLE
Error while handling slice operation

### DISCUSSION THREAD

**Discussion by superpi15** [Xilinx/finn#479] [Q&A]
Hi friends, 
First of all, thank you for the great work. 
I've tired various things and were able to deploy to boards. I enjoy it a lot. 
Today I tried to deploy an object detection model with the end2end notebook. 
I encountered a problem in the section shown here. It reads in yolo_4b.onnx for preprocessing. 
![image](https://user-images.githubusercontent.com/6512963/147337421-221e589e-da4f-43a8-be65-273997469442.png)
It reports the model is invalid, as shown in the following screenshot. 
![image](https://user-images.githubusercontent.com/6512963/147337455-28965c90-f4a8-4dba-a749-926599d70883.png)
To my knowledge "slice" operation should accepts 5 inputs. 
But the error message says min / max input should be within [1,1], that is, only one input is allowed. 

Any hint, ideas, or experience sharing are welcome. 

Thanks, 
Nero Horus

**Comment by patrickgeel** [comment]
I have run into this same error and believe the issue to be that in the newer version of ONNXRUNTIME, the "starts, axes, ends, and split" need to be passed as an attribute instead of an input.
